### ISSUE TITLE
Add `strip-gotcha` tag

### ISSUE THREAD

**Issue by Shom770** [python-discord/meta#156]
### Tag Name

strip/strip-gotcha

### What kind of content should the tag include?

When working with `strip`, `lstrip`, or `rstrip`, you might think that this would be the case:
```py
"Monty Python".rstrip(" Python")
>>> "Monty"
```
While this seems intuitive, it would actually result in:
```py
"Monty Python".rstrip(" Python")
>>> "M"
```
as Python interprets the argument to these functions as a set of characters rather than a substring.

If you want to remove a prefix/suffix from a string, `​str.removeprefix` and `str.removesuffix` are recommended and were added in 3.9.
```py
"Monty Python".removesuffix(" Python")
>>> "Monty"
```
See the documentation for `str.removeprefix` and `str.removesuffix` [here](https://docs.python.org/3/library/stdtypes.html#str.removeprefix)

**Comment by greyblue9** [comment]
This is a useful tag. I was surprised by this myself once.

**Comment by HassanAbouelela** [comment]
Merged in python-discord/bot#2000
